### PR TITLE
Minifier script

### PR DESCRIPTION
**Pull request by sczembor**
- add two scripts 
- add readme

[details](https://docs.near.org/sdk/rust/building/post-processing)

### CODE REVIEW

File: contracts/minifier/minify_contracts.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+pushd ..
+make build
+popd
+mkdir -p ./out/base/
+cp ../res/* ./out/base/
```
Comment (robert-zaremba) [inline contracts/minifier/minify_contracts.sh:6]: let's put the files in `/res-min`.

File: contracts/minifier/minify_contracts.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#for p in /work/near/core-contracts/*/res/*.wasm ; do
+#  cp $p ./out/base/
+# done
```
Comment (robert-zaremba) [inline contracts/minifier/minify_contracts.sh:9]: let's remove the comment.

**Review by robert-zaremba — APPROVED** [review]
pre-approved

File: contracts/minifier/minify_contracts.sh (hunk 1 above)
Comment (sczembor) [reply contracts/minifier/minify_contracts.sh]: DONE

File: contracts/minifier/minify_contracts.sh (hunk 2 above)
Comment (sczembor) [reply contracts/minifier/minify_contracts.sh]: DONE

**Pull request merged**